salt/metalk8s/salt/minion/running.sls

```diff
@@ -5,7 +5,7 @@ Restart salt-minion:
 Wait until salt-minion restarted:
   test.configurable_test_state:
     - changes: False
-    - result: __slot__:salt:test.sleep(10)
+    - result: __slot__:salt:test.sleep(20)
```

Comment:
Never fond of these, bumping timeouts is cool, because it only has a positive impact in worse scenarios, but bumping a sleep also has a negative impact in better scenarios.
Are you sure that's all we can do for this?
What if we had the minion write/touch a file on startup (maybe there's already something in place?), and we waited for such a file to be updated (with a timeout :wink:)?

Reply:
Removed, you right it's not really make sense to bump this one here. Would need more investigation